olivia/histos.py

```diff
@@ -1,11 +1,11 @@
 import numpy  as np
 import tables as tb
 
-from .. reco  import tbl_functions as tbl
+from invisible_cities.reco  import tbl_functions as tbl
 
 
 class Histogram:
-    def __init__(self, title, bins, labels, values=None):
+    def __init__(self, title, bins, labels, scale, values=None):
```

Comment:
It's not entirely clear to me what 'scale' does.

Is it related to the normalisation? Is it log vs. linear. I think a more descriptive name needs to be found

Reply:
scale is a Histogram atribute for plotting purpuses, it says how the Histogram y axis should be plotted and do not affect the Histogram class by itself. This behaviour is the same as title and labels atributes, which are just for identification and plotting purpuses. Scale should have one of the matplotlib allowed scales: linear, log, symlog, logit.

Note that the code is retrocompatible since this scale is set automatically to linear when importing old olivia histograms in the `get_histograms_from_file` function in hist_io module.

Reply:
It's still not clear what this does. Is it whether or not you normalise? It's a float so it's not just a decision between log or linear scale.

Reply:
I think I don't get your point. **scale** is not a float, it is a string with the posible matplotlib allowed scales linear, log, symlog, logit. The normalization is set to True in histogram_plot_functions.py. 

In fact this is something that I talk with @Aretno to modify because this normalizes the histogram integral, not the bin values. But finally we let this as it was.

Reply:
In the docstring it is described as a float and since it's not obvious where it's used I couldn't contradict this.
Do a PR correcting the error, please.